src/components/SpaceProposalInformation.vue

```diff
@@ -34,13 +34,7 @@ const symbols = computed((): string[] =>
             :key="symbol"
             class="flex"
           >
-            <span
-              v-tippy="{
-                content: symbol
-              }"
-            >
-              <AvatarSpace :space="space" :symbol-index="symbolIndex" />
-            </span>
+            <AvatarSpace :space="space" :symbol-index="symbolIndex" />
```

Comment:
Now tooltip doesn't work for me, should be fine?

Reply:
Yeah I removed it, doesn't seem to be needed